handler/handler_test.go

```diff
@@ -139,23 +132,23 @@ func TestNew(t *testing.T) {
 					}
 
 					var key, want string
-					key, want = "X-Athenz-Principal", rt.Principal
+					key, want = "X-Athenz-Principal", "rt_principal"
 					if err := f(key, want); err != nil {
 						return err
 					}
-					key, want = "X-Athenz-Role", strings.Join(rt.Roles, ",")
+					key, want = "X-Athenz-Role", "rt_role1,rt_role2,rt_role3"
 					if err := f(key, want); err != nil {
 						return err
 					}
-					key, want = "X-Athenz-Domain", rt.Domain
+					key, want = "X-Athenz-Domain", "rt_domain"
 					if err := f(key, want); err != nil {
 						return err
 					}
-					key, want = "X-Athenz-Issued-At", strconv.FormatInt(rt.TimeStamp.Unix(), 10)
+					key, want = "X-Athenz-Issued-At", strconv.FormatInt(1595908257, 10)
```

Comment:
```suggestion
					key, want = "X-Athenz-Issued-At", "1595908257"
```
please also fix other `strconv.FormatInt()`

Reply:
Fixed
https://github.com/yahoojapan/authorization-proxy/pull/66/commits/4459dce9813b711d85e81e3b494dc339d4c371f2